packages/deployment/files.js

```diff
@@ -31,9 +33,7 @@ const fsClose = util.promisify(rawClose);
 export { resolve, dirname, basename };
 
 export const needNotExists = async filename => {
-  if (await exists(filename)) {
-    throw Error(`${filename} already exists`);
-  }
+  assert(!(await exists(filename)), d`${filename} already exists`);
```

Comment:
violates our `await` rule

Reply:
Wow, a bunch of them.

Reply:
Actually, only one within an `assert`. But searching for `(await` turns up 50 hits!

Reply:
https://github.com/Agoric/agoric-sdk/issues/2416